example/example.go

```diff
@@ -20,9 +20,9 @@ const (
 )
 
 func downloadISO(fcosstable stream.Stream) error {
-	iso := fcosstable.Architectures[targetArch].Artifacts["metal"].Formats["iso"].Disk
-	if iso == nil {
-		return fmt.Errorf("%s: missing iso", fcosstable.FormatPrefix(targetArch))
+	iso, err := fcosstable.QueryDisk(targetArch, "metal", "iso")
+	if err != nil {
+		return err
```

Comment:
Heh, I was about ask why you weren't using `?` until I realized I wasn't reading Rust.